src/current/_includes/v24.3/resilience/dr-feature-table.md

```diff
@@ -0,0 +1,64 @@
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Point-in-time backup & restore</th>
+    <th>Physical cluster replication (asynchronous)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
```

Comment:
It would be helpful to define RPO and RTO on this page, perhaps in the paragraph leading into this tabel.

Reply:
+1

Reply:
Yes, they were defined on the Disaster Recovery overview page, but I've added to an include and popped the definitions on this page too.